plugins/modules/inventory_intent.py

```diff
@@ -2774,37 +2851,39 @@ def verify_diff_merged(self, config):
         if device_added:
             if not devices_to_add:
                 self.status = "success"
-                msg = "Requested Devices - {0} Added in Cisco DNA Center and Addition verified.".format(str(device_ips))
-                self.log(msg)
+                msg = "Requested Device(s) '{0}' Added in Cisco Catalyst Center and Addition verified.".format(str(device_ips))
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device addition task not executed successfully.")
+                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device addition task not executed successfully.", "INFO")
 
         if device_updated and self.config[0].get('update_interface_details'):
             interface_update_flag = True
+            interface_name = self.config[0].get('update_interface_details').get('interface_name')
 
             for device_ip in device_ips:
-                if not self.check_interface_details(device_ip):
+                if not self.check_interface_details(device_ip, interface_name):
                     interface_update_flag = False
                     break
 
             if interface_update_flag:
                 self.status = "success"
                 msg = "Interface details updated and verified successfully for devices {0}.".format(device_ips)
-                self.log(msg)
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning update interface details task not executed successfully.")
+                self.log("""Playbook parameter does not match with Cisco Catalyst Center, meaning update interface details
+                         task not executed successfully.""", "INFO")
 
         if device_updated and credential_update and device_type == "NETWORK_DEVICE":
             credential_update_flag = self.check_credential_update()
 
             if credential_update_flag:
                 self.status = "success"
                 msg = "Device credentials and details updated and verified successfully in Cisco Catalyst Center."
-                self.log(msg)
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device updation task not executed properly.")
+                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device updation task not executed properly.", "INFO")
         elif device_type != "NETWORK_DEVICE":
-            self.log("Cannot compare the parameter for device type {0} in the playbook with Cisco Catalyst Center.".format(device_type))
+            self.log("Cannot compare the parameter for device type {0} in the playbook with Cisco Catalyst Center.".format(device_type), "WARNING")
```

Comment:
self.log("Unable to compare the parameter for device type '{0}' in the playbook with the one in Cisco Catalyst Center.".format(device_type), "WARNING")

Reply:
added.